lib/cryptoexchange/exchanges/graviex/graviex.yml

```diff
@@ -0,0 +1,68 @@
+:pairs:
```

Comment:
Since they have an API returning list https://graviex.net/api/v2/markets.json
That would be preferred over manual yml @demuskov

Reply:
yml file not needed - i'v rewrite code for direct pairs listing from API